clang-tools-extra/test/clang-tidy/checkers/bsl-destructor-access-specifier.cpp

```diff
@@ -0,0 +1,72 @@
+// RUN: %check_clang_tidy %s bsl-destructor-access-specifier %t
+
+class A
+{
+  public:
+    ~A() // Non-compliant
```

Comment:
Same thing here. Missing the LLVM comments to detect the cases that should fail.